aggregate/site/category.xml

```diff
@@ -284,10 +284,20 @@
     <category name="CoreTools" />
     <category name="MavenTools"/>
   </feature>
-  <feature id="org.eclipse.linuxtools.docker.feature">
+  <!-- JBDS-3495 add plugins instead of feature to avoid p2.inf instruction which forces JBDS to include Docker nightly update site in disabled state -->
```

Comment:
Because removing them from the update site doesn't remove them from the feature itself. My previous solution was incomplete and full o' fail. This one actually works.